src/update.ts

```diff
@@ -62,40 +62,44 @@ class DenoGraph {
 }
 
 export async function collect(
-  entrypoints: string | string[],
-  options: {
-    importMap?: string;
-  } = {},
+  entrypoints: {
```

Comment:
I'm afraid I don't love this change in the API.
Can we keep the original form and require users to call the function for each group of files which share an import map?

Reply:
I can keep it, but the share entry point part is not straightforward to do , I think the obvious way is to iterate over the entry points and call the original function on each

Reply:
Hmm.. let me create a PR for experiment and explanation

Reply:
> I think the obvious way is to iterate over the entry points and call the original function on each

This would also be fine for now, although it's very inefficient if they have overlaps on dependencies.

Reply:
done, I think the usual usage of molt ci is going to be with one entry-point, I doubt multiple entry-points is that frequent so I think it should be fine.

Reply:
The usual usage for me is being with multiple "entrypoints" actually 🤣 Maybe how I manage dependencies is just too tricky.
Thanks anyways.